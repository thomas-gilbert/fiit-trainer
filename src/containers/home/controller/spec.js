import React from 'react';
import { Controller } from './index.js';

describe('Controller Component', () => {
  it('renders correctly', () => {
    const props = {
      exercises: {
        cardio: [],
        weights: [],
      },
      getCardioExercisesAsync: jest.fn(),
      getWeightExercisesAsync: jest.fn(),
    };
    const component = global.shallow(
      <Controller {...props} />
    );

    expect(component).toMatchSnapshot();
  });
});
