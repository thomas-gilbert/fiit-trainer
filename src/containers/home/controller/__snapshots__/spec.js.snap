// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Controller Component renders correctly 1`] = `
<div>
  <Row
    bsClass="row"
    componentClass="div"
  >
    <Col
      bsClass="col"
      componentClass="div"
      xs={6}
      xsOffset={6}
    >
      <label>
        Workout Time
        <input
          defaultValue="20"
          placeholder=""
          type="number"
        />
        Minutes
      </label>
      <br />
      <label>
        Workout Interval
        <input
          defaultValue="40"
          placeholder=""
          type="number"
        />
        Seconds
      </label>
      <br />
      <label>
        Rest Interval
        <input
          defaultValue="20"
          placeholder=""
          type="number"
        />
        Seconds
      </label>
      <br />
      <button>
        START
      </button>
    </Col>
  </Row>
  <Row
    bsClass="row"
    componentClass="div"
  >
    <Col
      bsClass="col"
      componentClass="div"
      xs={6}
    >
      <h3>
        Cardio Exercises
      </h3>
      <Connect(ExerciseSelect)
        defaultValue="Select exercise"
        options={Array []}
      />
    </Col>
    <Col
      bsClass="col"
      componentClass="div"
      xs={6}
    >
      <h3>
        Weight Exercises
      </h3>
      <Connect(ExerciseSelect)
        defaultValue="Select exercise"
      />
    </Col>
  </Row>
</div>
`;
